source/gui/__init__.py

```diff
@@ -50,11 +50,13 @@
 
 
 if version_year < 2023:
-	from .exit import quit  # noqa F401: imported for backward compatibility
-	"""
-	Deprecated, for removal in 2023.
-	Use L{gui.exit.quit} instead.
-	"""
+	def quit():
+		"""
+		Deprecated, for removal in 2023.
+		Use `wx.CallAfter(mainFrame.onExitCommand, None)` directly instead.
+		"""
+		wx.CallAfter(mainFrame.onExitCommand, None)

```

Comment:
Can you add a log message (perhaps debug warning?) that this is deprecated. Perhaps we should add a category specifically for deprecations.

Reply:
This has been added in [22d4cc0](https://github.com/nvaccess/nvda/pull/13498/commits/22d4cc061edf6609d27fa18a65efdcc21ee03a09)